### PR TITLE
Enable creating folders for experiments and improve naming convention

### PR DESCRIPTION
**Pull request by avv-va**

### CODE REVIEW

File: oai_agents/agents/rl.py [hunk 1]
```diff
@@ -251,8 +249,7 @@ def wrap_agent(self, sb3_agent, name):
         return SB3Wrapper(sb3_agent, name, self.args)
 
     def get_experiment_name(self, exp_name):
-        all_train_teamtypes = [tag for tags_dict in self.teammates_collection.values() for tag in tags_dict.keys()]
-        return exp_name or 'train_' + '_'.join(all_train_teamtypes)
+        return exp_name or str(self.args.exp_dir) + '/' + self.name
```
Comment (Jmiceli17) [inline oai_agents/agents/rl.py:252]: recommend putting parentheses around `or` statement to ensure it evaluates how intended
```
return (exp_name or str(self.args.exp_dir)) + '/' + self.name
```

**Review by Jmiceli17 — COMMENTED** [review]
Looks good! Just one small recommendation

File: oai_agents/agents/rl.py (hunk 1 above)
Comment (avv-va) [reply oai_agents/agents/rl.py:252]: this was intentional because sometimes we may want to override the whole experiment_name :D

Comment (Jmiceli17) [reply oai_agents/agents/rl.py:252]: ah whoops! That makes sense 😅

File: scripts/utils/train_helper.py [hunk 2]
```diff
@@ -176,8 +200,13 @@ def get_fcp_trained_w_selfplay_types(args,
                                                         train_types=fcp_w_sp_train_types,
                                                         eval_types=fcp_w_sp_eval_types,
                                                         )
-
-    name = 'fcp_w_selfplay_types'
+    seed, h_dim = 2602, 256
+    name = generate_name(args,
```
Comment (Ttopiac) [inline scripts/utils/train_helper.py:204]: beautiful function!

**Pull request merged**